security/acme-client/src/opnsense/mvc/app/library/OPNsense/AcmeClient/LeAutomation/ConfigdRemoteSsh.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * Copyright (C) 2020-2021 Frank Wall
```

Comment:
Please change this to your name and the current year. Nit-picking, but stil... :)

Reply:
Sure, since I just copied the file I left it  as-is, but will update it.